lib/src/pages/layouts/yaru_master_detail_page.dart

```diff
@@ -32,6 +34,9 @@ class YaruMasterDetailPage extends StatefulWidget {
     required this.titleBuilder,
     required this.pageBuilder,
     required this.leftPaneWidth,
+    this.leftPaneResizing = true,
+    this.leftPaneMinWidth = 175.0,
+    this.pageMinWidth = 310.0,
```

Comment:
hmm, these are a bit arbitrary numbers :)

Reply:
Indeed, but we need ones :)

At 175 width, the left pane only show a few letters, 150 is imo too few.
And 310, is the half of the width (620) where the layout switch to portrait layout.

But I'm open to some other values or other way do add defaults :)

Reply:
Depends also on the system font. What about moving that `620` to `constants.dart` and using `kFoo / 2`?

Reply:
The `175` is totally arbitrary, I admit I set this value just because I had feel it was the best one.
But honestly, this value is kind of edge use case. It looks good "by default", when an user want to collapse the sidebar at maximum. This is just prevent any widget to overflow and broke the layout. If the app is define a tile which can't be collapsed so much, he can overwrite this value.
At least it is how I see this value :)

About the `620` one, I see you created a `YaruMasterDetailThemeData` with this value hardcoded as fallback.
So, should we use the same constant there?

Reply:
sharing the same constant would help to make the intention/origin clear. when i saw 310 at first, i didn't realize it was the half. 😁

Reply:
Btw, I wrote `kFoo` because I have no idea what the constant should be called. Do you have something in mind?

Reply:
What about something like simply `kMasterDetailBreakpoint`?

Reply:
Sure, I prefixed them all with `kYaru...` to distinguish them from Flutter's constants, though.

Reply:
Okay, so let's do `kYaruMasterDetailBreakpoint` :)